--- .speech-dispatcher/conf/speechd.conf 
+++ open-sapi/packages/speech-dispatcher/speechd.conf 
@@ -36,9 +36,10 @@
 # DO NOT COMMENT OUT THIS OPTION, SET IT TO "default" if you do not
 # want to influence it.
 
-LogDir  "default"
+#LogDir  "default"
+#LogDir "/homw/tom/.speech-dispatcher/conf/session.log
 #LogDir  "/var/log/speech-dispatcher/"
-#LogDir  "stdout"
+LogDir  "stdout"
 
 # The CustomLogFile allows logging all messages of the given kind,
 # regardless their priority, to the given destination.
@@ -206,10 +207,10 @@
 #    either relative (to etc/speech-dispatcher/modules/) or absolute
 
 AddModule "espeak"       "sd_espeak"   "espeak.conf"
-AddModule "festival"     "sd_festival"  "festival.conf"
-AddModule "flite"        "sd_flite"     "flite.conf"
+AddModule "festival"     "sd_festival" "festival.conf"
+AddModule "flite"        "sd_flite"    "flite.conf"
 AddModule "ivona"	 "sd_ivona"    "ivona.conf"
-#AddModule "espeak-generic" "sd_generic" "espeak-generic.conf"
+AddModule "open-sapi-generic"    "sd_generic"  "open-sapi-generic.conf"
 #AddModule "espeak-mbrola-generic" "sd_generic" "espeak-mbrola-generic.conf"
 #AddModule "swift-generic" "sd_generic" "swift-generic.conf"
 #AddModule "epos-generic" "sd_generic"   "epos-generic.conf"
